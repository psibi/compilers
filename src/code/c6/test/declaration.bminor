b: boolean;
str: string;
i: integer;
