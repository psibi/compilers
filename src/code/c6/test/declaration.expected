b (boolean)
str (string)
i (integer)
i2 (integer)(3)
